### PR TITLE
[FIX] fleet: fix fuel types

### PR DESCRIPTION
**Pull request by Bertrand2**
There shouldn't be a distinction between full hybrids.
'hybrid' and 'full_hybrid_gasoline' are replaced in this commit by a unique 'full_hybrid'.
fixes task 2629318

taskID 2753096

### CODE REVIEW

File: addons/fleet/i18n/fleet.pot [hunk 1]
```diff
@@ -906,15 +906,9 @@ msgid "Horsepower Taxation"
...
-msgid "Hybrid Diesel"
-msgstr ""
-
-#. module: fleet
-#: model:ir.model.fields.selection,name:fleet.selection__fleet_vehicle__fuel_type__full_hybrid_gasoline
-#: model:ir.model.fields.selection,name:fleet.selection__fleet_vehicle_model__default_fuel_type__full_hybrid_gasoline
-msgid "Hybrid Gasoline"
+#: model:ir.model.fields.selection,name:fleet.selection__fleet_vehicle__fuel_type__full_hybrid
+#: model:ir.model.fields.selection,name:fleet.selection__fleet_vehicle_model__default_fuel_type__full_hybrid
+msgid "Full Hybrid"
```
Comment (kba-odoo) [inline addons/fleet/i18n/fleet.pot:911]: You can undo this change :-)

Comment (Bertrand2) [reply addons/fleet/i18n/fleet.pot]: Indeed! It is removed.

**Comment by kba-odoo** [comment]
robodoo r+

**Comment by kba-odoo** [comment]
robodoo retry

**Comment by tivisse** [comment]
FYI: https://github.com/odoo/enterprise/pull/27315/commits/2d8a3304dff671d1583e1f10a658fe1454c36e17
